### PR TITLE
[Bug Fix]LPS-171752 Add new job scheduler trigger button is missing because of uncaught type error

### PR DESCRIPTION
**Pull request by W2jx**
see https://issues.liferay.com/browse/LPS-171752

### CODE REVIEW

**Comment by joelrocha68** [comment]
Just starting reviewing :)

**Comment by zhangyan0701** [comment]
ci:test:data-integration

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by tinatian** [comment]
@catarineaguiar do the ci test we triggered covered the case the ticket mentioned?

Tina

**Comment by tinatian** [comment]
see https://github.com/liferay-core-infra/liferay-portal/pull/2905

File: modules/apps/dispatch/dispatch-service/src/main/java/com/liferay/dispatch/internal/executor/DispatchTaskExecutorRegistryImpl.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ public boolean isClusterModeSingle(String type) {
 			return dispatchTaskExecutor.isClusterModeSingle();
 		}
 
-		return false;
+		return true;
```
Comment (tinatian) [inline modules/apps/dispatch/dispatch-service/src/main/java/com/liferay/dispatch/internal/executor/DispatchTaskExecutorRegistryImpl.java:80]: @W2jx I do not understand why you made this change

**Pull request closed without merging**